src/central_control/scripts/central_control.py

```diff
@@ -14,7 +14,7 @@ class CentralControl:
     def __init__(self):
         host_ip = rospy.get_param('/host_ip')
 
-        self.client = airsim.CarClient(ip = host_ip)
```

Comment:
This was getting really annoying (preventing ROS from connecting to AirSim), and I'm not sure any of us actually need this IP variable